[SQUASH] base: Tint QS Tile with accent color [1/2]

Laker87 opened the pull request:
* Tinted QS Tile stuffs (icon, background, label)
* Using bolder font

Adjust QS tile background and icon color

+ make qs tile icon smaller by 1dp
+ reduce qs tile vertical margin

SystemUI: Add a toggle for the new QS tint [1/2]

Co-authored-by: Stallix <joey@evolution-x.org>
Signed-off-by: Joey Huab <joey@evolution-x.org>

neobuddy89 commented on packages/SystemUI/src/com/android/systemui/qs/tileimpl/QSTileImpl.java:
```diff
@@ -538,14 +539,24 @@ public String getMetricsSpec() {
+                if (setQsUseNewTint) {
+                    return Utils.getColorAttrDefaultColor(context, android.R.attr.textColorSecondary);
+                } else {
+                    return Utils.getColorAttrDefaultColor(context, android.R.attr.textColorSecondary);
```
what's different here? if and else has same line of codes.

neobuddy89 commented on packages/SystemUI/res/values/styles.xml:
```diff
@@ -217,7 +217,7 @@
     <style name="TextAppearance.QS.TileLabel">
         <item name="android:textSize">@dimen/qs_tile_text_size</item>
-        <item name="android:fontFamily">@*android:string/config_headlineFontFamily</item>
+        <item name="android:fontFamily">@*android:string/config_headlineFontFamilyMedium</item>
```
why changing style?

neobuddy89 commented on packages/SystemUI/res/values/dimens.xml:
```diff
@@ -495,7 +495,7 @@
     <dimen name="qs_tile_margin_horizontal">18dp</dimen>
     <dimen name="qs_tile_margin_horizontal_two_line">2dp</dimen>
-    <dimen name="qs_tile_margin_vertical">2dp</dimen>
+    <dimen name="qs_tile_margin_vertical">16dp</dimen>
```
why changing margin?

neobuddy89 commented on core/java/android/provider/Settings.java:
```diff
@@ -5541,6 +5547,7 @@ public static void setShowGTalkServiceStatusForUser(ContentResolver cr, boolean
             PRIVATE_SETTINGS.add(LOCK_TO_APP_ENABLED);
             PRIVATE_SETTINGS.add(EGG_MODE);
             PRIVATE_SETTINGS.add(DISPLAY_COLOR_MODE);
+            PRIVATE_SETTINGS.add(QS_PANEL_BG_USE_NEW_TINT);
```
remove PRIVATE_SETTINGS

neobuddy89 commented on packages/SystemUI/src/com/android/systemui/qs/tileimpl/QSTileBaseView.java:
```diff
@@ -125,8 +129,16 @@ public QSTileBaseView(Context context, QSIconView icon, boolean collapsedView) {
-                Utils.getColorAttrDefaultColor(context, android.R.attr.textColorTertiary));
+        mColorActiveAlpha = adjustAlpha(mColorActive, 0.2f);
+        boolean setQsUseNewTint = Settings.System.getIntForUser(context.getContentResolver(),
+                Settings.System.QS_PANEL_BG_USE_NEW_TINT, 1, UserHandle.USER_CURRENT) == 1;
```
should not be enabled by default.

00day0 committed:
[SQUASH] base: Tint QS Tile with accent color [1/2]

* Tinted QS Tile stuffs (icon, background, label)
* Using bolder font

Adjust QS tile background and icon color

+ make qs tile icon smaller by 1dp
+ reduce qs tile vertical margin

SystemUI: Add a toggle for the new QS tint [1/2]

Co-authored-by: Stallix <joey@evolution-x.org>
Signed-off-by: Joey Huab <joey@evolution-x.org>

Laker87 commented:
Fixed

neobuddy89 commented:
Added.
https://github.com/crdroidandroid/android_frameworks_base/commit/bab41b028058e33f54dbd606e6f0e46b36f97b1d
